src/reactor/poll_evented.rs

```diff
@@ -17,6 +17,12 @@ use tokio_io::{AsyncRead, AsyncWrite};
 
 use reactor::{Handle, Direction};
 
+struct PollEventedInner {
```

Comment:
I've been meaning to make this change.

Would you mind renaming this struct to `Registration` and updating the `PollEvented` field name to `registration` as well?